client/ayon_core/hosts/max/api/lib.py

```diff
@@ -221,18 +226,15 @@ def reset_scene_resolution():
     scene resolution can be overwritten by a folder if the folder.attrib
     contains any information regarding scene resolution.
     """
-
-    folder_entity = get_current_project_folder(
-        fields={"attrib.resolutionWidth", "attrib.resolutionHeight"}
-    )
-    folder_attributes = folder_entity["attrib"]
-    width = int(folder_attributes["resolutionWidth"])
-    height = int(folder_attributes["resolutionHeight"])
+    task_entity = get_current_task_entity()
+    task_attributes = task_entity["attrib"]
+    width = int(task_attributes["resolutionWidth"])
+    height = int(task_attributes["resolutionHeight"])
 
     set_scene_resolution(width, height)
 
 
-def get_frame_range(folder_entiy=None) -> Union[Dict[str, Any], None]:
+def get_frame_range(task_entity=None) -> Union[Dict[str, Any], None]:
     """Get the current folder frame range and handles.
 
     Args:
```

Comment:
Docstring needs updating with the argument change.

Reply:
yes, and I also find out that the task entity you suggested in other maya PRs is already getting attributes in the dict I assumed